MaterialLib/MPL/PropertyType.h

```diff
@@ -311,7 +311,7 @@ static const std::array<std::string, PropertyType::number_of_properties>
                              "residual_gas_saturation",
                              "residual_liquid_saturation",
                              "retardation_factor",
-                             "saturation",
+                             "saturation_capillary_pressure",
```

Comment:
This is now ambiguous. Would this return saturation or pressure? Or both?
If there is need for capillary pressure model (*e.g.* from liquid saturation), I'd recommend to add a new property `capillary_pressure`.

Reply:
Yes it is ambiguous. If the property name refers to the meaning the returned value, `saturation` is a proper name. Then all properties that give saturation is covered by one property regardless their physical property.  Anyway, I am going to remove the commit of renaming.

Reply:
Let's try the simple way of new `capillary_pressure` property with a single `CapillaryPressureVanGenuchten` model. Then, when it is working together with H2-p-rho, we discuss how to generalize. What do you think?

Reply:
Agree. The commits of renaming have been removed.